cms/djangoapps/api/v1/tests/test_views/test_course_runs.py

```diff
@@ -167,9 +168,9 @@ def test_partial_update(self):
         url = reverse('api:v1:course_run-detail', kwargs={'pk': str(course_run.id)})
         response = self.client.patch(url, data, format='json')
         assert response.status_code == 200
-        self.assert_access_role(course_run, existing_user, role)
-        self.assert_access_role(course_run, new_user, role)
-        self.assert_course_access_role_count(course_run, 2)
+        self.assert_access_role(course_run, existing_user, staff_role)
+        self.assert_access_role(course_run, new_user, instructor_role)
+        self.assert_course_access_role_count(course_run, 3)
```

Comment:
Why it is 3? we already have `staff_role` for `existing_user`, after adding `instructor_role ` shouldn't it be 2?
Am I missing something?

Reply:
the new_user has multiple roles now because we are using get_or_create method. new_user has both the staff_role and the instructor_role

Reply:
Got that. Thank you.